backend/flask_api.py

```diff
@@ -8,7 +8,6 @@
 import time
 import json
 import json5
-
```

Comment:
??? please keep that in, the encouraged form is:

-absolute imports- (import library)

-relative imports- (from library import X)

-own absolute imports- (import projectfile)
-own relative imports- (from projectfile import X)